### PR TITLE
Portfolio: mobile version

### PR DESCRIPTION
**Pull request by Bondok6**
Added two sections 
1. Work section
2. About section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -63,12 +63,219 @@ <h1 class="hero__title">Hey there. I’m Kyrillos <span>I’m a software develop
...
+            <img src="images/work-screenshots/Bankist.PNG" alt="bankist-app" class="card__img">
+          </div>
+
+          <div class="card__details">
+            <h3 class="card__text">bankist website</h3>
+
+            <ul class="card__list">
+              <li>javaScript</li>
+              <li>css</li>
+              <li>React</li>
```
Comment (andersonlebon) [inline index.html:88]: - [x] According to the requirement the project(card) should be` <li>` and inside a `<ul>` tag. 
- Your implementation using `articles` tags
```
 <div class="card-container">
      <article class="card"> </article>
      <article class="card"></article>
      <article class="card"></article>
      <article class="card"></article>
      <article class="card">
</div>
```
- A better implementation using `ul` tag

```
 <ul class="card-container">
      <li class="card"></li>
      <li class="card"></li>
      <li class="card"></li>
      <li class="card"></li>
      <li class="card"></li>
</ul>
```

File: styles/main.css [hunk 2]
```diff
@@ -107,3 +161,140 @@ header {
...
+}
+
+.card__list,
+.about__card__list {
+  width: 100%;
+  display: -webkit-box;
+  display: flex;
+  justify-content: space-evenly;
+  flex-wrap: wrap;
+}
```
Comment (andersonlebon) [inline styles/main.css:243]: - [x] Compare your design on the right side and the template one on the left side then:
- Reduce the space between technologies(ruby on rails, bootstrap, HTML, CSS...) by changing the `justify-content: space-evenly` with `justify-content: space-center`

Screenshot from 2021-11-26 03-57-17

**Review by andersonlebon — CHANGES_REQUESTED** [review]
Hi @Bondok6 

Good job so far! I know you have worked hard on making this milestone But  There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- You  are following GitHub actions :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Good use of `flex` and `grid` properties :heavy_check_mark: 
- Your design looks perfect :100: :+1:  :heavy_check_mark: 
## Required Changes ♻️

- [ ] Kindly check comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Cathella — APPROVED** [review]
## STATUS: APPROVED 🟢

### Hi @Bondok6 👋🏻, 

You have done well implementing the changes requested by the previous reviewer. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Suggestions 

- [ ] [Optional] 🔔 Please capitalize commit messages to make them look professional 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**